src/node.h

```diff
@@ -321,6 +321,13 @@ enum IsolateSettingsFlags {
   SHOULD_NOT_SET_PREPARE_STACK_TRACE_CALLBACK = 1 << 3
 };
 
+enum InitializationSettingsFlags : uint64_t {
+  kDefaultInitialization = 1 << 0,
+  kInitializeV8 = 1 << 1,
+  kRunPlatformInit = 1 << 2,
+  kInitOpenSSL = 1 << 3
+};
+
```

Comment:
Since `InitializeOncePerProcess` is internal for now, I’d also put this into `node_internals.h` (or move both into `node.h` and document them)